.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,38 @@
+name: Hextractor build and test
+
+on: [push]
```

Comment:
I would rather change it to `on: [pull_request]` as `on: [push]` will run when a push is made to any branch in the workflow's repository,